schema/src/cwms/cwms_text_pkg_body.sql

```diff
@@ -204,7 +204,7 @@ as
    begin
       cwms_util.check_office_permission(p_office_id);
 
-      if p_text is null or dbms_lob.getlength(p_text) = 0 then
+      if p_text is null then
```

Comment:
@perrymanmd @Enovotny 


Should we allow NULL? if there's no text why even bother creating a clob in the first place?

At same time there is a difference between "there is nothing" and "there is something but it's empty"

Reply:
Updated tp remove null check as well based on decision during db meeting